src/components/NumericInput/NumericInput.vue

```diff
@@ -177,13 +178,18 @@ function convertFromDisplayValue(v) {
   return value;
 };
 
-const inputSpinnerClass = props.noSpinner
-  ? '[-moz-appearance:_textfield] [&::-webkit-outer-spin-button]:m-0 [&::-webkit-outer-spin-button]:appearance-none [&::-webkit-inner-spin-button]:m-0 [&::-webkit-inner-spin-button]:appearance-none'
-  : '';
+const enableSpinner = !(props.spinner===undefined ||  props.spinner===null) ? props.spinner : useDefaultSpinner(props);
```

Comment:
Minor but the suggestion below might be a bit tidier
```suggestion
const enableSpinner = (typeof props.spinner === 'boolean') ? props.spinner : useDefaultSpinner(props);
```